lib/features/create_account/create_account.dart

```diff
@@ -141,8 +160,34 @@ class _CreateAccountState extends State<CreateAccount> {
                         showSpinner = true;
                       });
 
+                      if (email.isEmpty || password.isEmpty) {
+                        setState(() {
+                          showSpinner = false;
+                        });
+                        showDialog(
+                          context: context,
+                          builder: (_) => AlertDialog(
+                            title: Text('Error'),
+                            content: Text('Debe rellenar los campos primero'),
+                            actions: [
+                              FlatButton(
+                                child: Text('OK'),
+                                onPressed: () {
+                                  Navigator.pop(context);
+                                },
+                              )
+                            ],
+                            elevation: 24.0,
+                            backgroundColor: Theme.of(context).primaryColor,
+                          ),
+                          barrierDismissible: false,
+                        );
+                      }
+
                       var res = await postRegister(email, password);
 
+                      var bodyRes = jsonDecode(res.body);
```

Comment:
```suggestion
                      MAP bodyRes = jsonDecode(res.body);
```
Creo deberiamos comenzar a trabajar con datos tipados y no con var, en todo caso si es que el cambio afecta la funcionalidad podemos dejarlo como `var` y trabajarlo mas adelante

Reply:
Dale, si es mejor